### PR TITLE
Fix `TestCheckHandlers` on Windows

### LINKED ISSUE #976: Fix path resolution on windows
https://github.com/valory-xyz/open-autonomy/blob/e4eb69e4c9b66fdaeddc9f8b4703a6a569534a2f/tests/test_autonomy/test_cli/test_analyse/test_abci/test_check_handlers.py#L35

### PR DESCRIPTION
**Pull request by Adamantios**
## Proposed changes

Fixes path resolution on Windows.

## Fixes

Fixes #976.

## Types of changes

What types of changes does your code introduce? (A **breaking change** is a fix or feature that would cause existing functionality and APIs to not work as expected.)
_Put an `x` in the box that applies_

- [ ] Non-breaking fix (non-breaking change which fixes an issue)
- [ ] Breaking fix (breaking change which fixes an issue)
- [ ] Non-breaking feature (non-breaking change which adds functionality)
- [ ] Breaking feature (breaking change which adds functionality)
- [ ] Refactor (non-breaking change which changes implementation)
- [ ] Messy (mixture of the above - requires explanation!)

## Checklist

_Put an `x` in the boxes that apply._

- [x] I have read the [CONTRIBUTING](../blob/main/CONTRIBUTING.md) doc
- [x] I am making a pull request against the `main` branch (left side). Also you should start your branch off our `main`.
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have locally run services that could be impacted and they do not present failures derived from my changes

## Further comments

n/a

### CODE REVIEW

**Comment by Adamantios** [comment]
Made a quick search for similar cases and found this:
https://github.com/valory-xyz/open-autonomy/blob/21b5ea131386a27593234b3b1bcaa8923d19f31d/scripts/generate_api_documentation.py#L94
Though the `create_subdir` method does not seem to be used.

File: tests/test_autonomy/test_cli/test_analyse/test_abci/test_check_handlers.py [hunk 1]
```diff
@@ -34,9 +34,6 @@
 from tests.test_autonomy.test_cli.base import BaseCliTest
 
 
-@pytest.mark.skipif(
-    platform.system() == "Windows", reason="Fix path resolutions on windows."
-)
 class TestCheckHandlers(BaseCliTest):
```
Comment (DavidMinarsch) [inline tests/test_autonomy/test_cli/test_analyse/test_abci/test_check_handlers.py:36]: What about `**/skill.yaml`?

**Review by DavidMinarsch — CHANGES_REQUESTED** [review]

File: tests/test_autonomy/test_cli/test_analyse/test_abci/test_check_handlers.py (hunk 1 above)
Comment (Adamantios) [reply tests/test_autonomy/test_cli/test_analyse/test_abci/test_check_handlers.py:36]: Isn't it safe since it is `pathlib`'s `Path.glob`?

**Review by 0xArdi — APPROVED** [review]

File: tests/test_autonomy/test_cli/test_analyse/test_abci/test_check_handlers.py (hunk 1 above)
Comment (DavidMinarsch) [reply tests/test_autonomy/test_cli/test_analyse/test_abci/test_check_handlers.py:36]: Dunno - can you confirm?

Comment (Adamantios) [reply tests/test_autonomy/test_cli/test_analyse/test_abci/test_check_handlers.py:36]: Just confirmed by using:
```python
import tempfile
from pathlib import Path


p = Path(tempfile.gettempdir())
print(list(p.glob("**/an_existing_folder_in_the_temp_dir")))
```

and it returned non-empty `list`.

**Review by DavidMinarsch — APPROVED** [review]

**Pull request merged**